lib/safe-pg-migrations/plugins/blocking_activity_logger.rb

```diff
@@ -78,6 +92,26 @@ def log_blocking_queries
       raise
     end
 
+    def output_blocking_queries(queries)
+      if SafePgMigrations.config.blocking_activity_logger_verbose
+        SafePgMigrations.say(
+          "Statement was being blocked by the following #{'query'.pluralize(queries.size)}:", true
+        )
```

Comment:
Could you please extract this one outside the `if`? It's relevant even if we don't display the query.